libraries/WiFi/src/WiFiProv.cpp

```diff
@@ -151,7 +151,9 @@ void WiFiProvClass :: beginProvision(scheme prov_scheme, wifi_prov_event_handler
         WiFi.mode(WIFI_MODE_STA);
         log_i("Aleardy Provisioned, starting Wi-Fi STA");
         log_i("CONNECTING ACCESS POINT CREDENTIALS : "); 
-        log_i("SSID : %s\n",WiFi.SSID().c_str());
+        wifi_config_t conf;
```

Comment:
you should check if the log level is info or higher, else this will cause unused variable and is unnecessary in reality.